docs/jax.scipy.rst

```diff
@@ -75,6 +75,8 @@ jax.scipy.linalg
    rsf2csf
    schur
    solve
+   solve_sylvester_eigen
+   solve_sylvester_schur
```

Comment:
Rather than having different functions, I think it would be better to have a single `solve_sylvester` function with a `method` parameter that lets you choose `method='eigen'` or `method='schur'`.